pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -4540,3 +4541,135 @@ func TestDisallowsInvalidFormatOptions(t *testing.T) {
 		}
 	}
 }
+
+type importSafeVisitor struct {
+	ctx    *tree.EvalContext
+	isSafe bool
+}
+
+func (v *importSafeVisitor) VisitPre(expr tree.Expr) (bool, tree.Expr) {
+	return v.isSafe, expr
+}
+
+var importSafeFunctions = map[string]struct{}{
+	"current_timestamp": {},
+	"now":               {},
+	"localtimestamp":    {},
+}
+
+func (v *importSafeVisitor) VisitPost(expr tree.Expr) (newNode tree.Expr) {
+	if fn, ok := expr.(*tree.FuncExpr); ok {
```

Comment:
I think you only need to look up into your map if the volatility is more than immutable right?

Reply:
Right -- and I already do that since this visitor is called after I check for that.

Reply:
No, you'll find functions here that pass the validator right? THe validator doesn't remove functions like `length()` that are immutable?